src/contrib/cluster/Akka.Cluster.Sharding/PersistentShard.cs

```diff
@@ -79,10 +79,11 @@ protected override void ProcessChange<T>(T evt, Action<T> handler)
         protected void SaveSnapshotIfNeeded()
         {
             PersistCount++;
-            if ((PersistCount & Settings.TunningParameters.SnapshotAfter) == 0)
+            if (PersistCount >= Settings.TunningParameters.SnapshotAfter)
```

Comment:
This will cause snapshots to be made on every even above configured threshold. The original intent was to make a snapshot every X persists, therefore it should be `%` operator instead of `>=`.